Global.cpp

```diff
@@ -90,7 +90,7 @@ void ErrorMessage(const char *msg)
 {
     #ifdef _WIN32
         std::wstring utf16String = ConvertToUtf16(msg, CP_UTF8);
-        MessageBoxW(NULL, utf16String.c_str(), L"Error", MB_ICONERROR);
+        MessageBoxW(NULL, utf16String.c_str(), L"Error", MB_ICONERROR | MB_SETFOREGROUND);
     #elif defined(__linux__)
         SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, "Error", msg, NULL);
```

Comment:
I'd like to find if there's a way do do the foreground in SDL but I don't know if there is. As SDL works cross-platform by MessageBoxW is Windows only so might be nice to clean this up at some point and also fix the bug cross-platform.